webpack.main.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         { from: 'source/main/modules/export/assets/export.tpl.htm', to: 'assets' },
         { from: 'source/main/modules/export/assets/template.revealjs.htm', to: 'assets' },
         { from: 'source/main/modules/export/assets/revealjs-styles', to: 'assets/revealjs-styles' },
-        { from: 'resources/icons/icon.ico', to: 'assets/icons' },
+        { from: 'source/main/modules/window-manager/assets/icons', to: 'assets/icons' },
```

Comment:
This change isn't need. Please remove the changes to `webpack.main.config.js` from the pull request.